ehabi32/ehabi32.rst

```diff
@@ -2652,9 +2652,21 @@ The encoding has been designed to meet the following criteria.
      address upon function exit. The Return Address Authentication Code is an
      unsigned integer with the same size as a general-purpose register. Upon
      completion of the unwinding instructions, the Return Address Authentication
-     Code can be used to validate the content of VRS[r15], using vsp as a
+     Code can be used to validate the content of VRS[r15] and if opcode 0xb5 is
+     encountered before completion of the unwinding, the effective VSP value at
+     code 0xb5 will be used as modifier otherwise the CFA is used as the
      modifier.
 
+  h) In a very small number of cases (for example, nested functions when a
+     closure pointer is passed as an additional argument), the inter-procedural
+     scratch register (IP) may be live on entry to the function.  In this
+     situation the return address cannot be signed until IP has been copied
+     somewhere else (for example, by saving it onto the stack or copy to
```

Comment:
"copy" -> "copying" to make it consistent

Reply:
Statement related to argument register is removed due to a downstream comment.